support-frontend/assets/helpers/subscriptionsForms/submit.js

```diff
@@ -80,42 +76,40 @@ function getAddresses(state: AnyCheckoutState) {
   };
 }
 
-const getOptions = (
-  fulfilmentOptions: FulfilmentOptions,
-  productOptions: ProductOptions,
-): {fulfilmentOptions: FulfilmentOptions, productOptions: ProductOptions} =>
-  ({
-    ...(fulfilmentOptions !== NoFulfilmentOptions ? { fulfilmentOptions } : {}),
-    ...(productOptions !== NoProductOptions ? { productOptions } : {}),
-  });
-
 const getProduct =
-  (state: AnyCheckoutState, currencyId?: Option<IsoCurrency>): DigitalSubscription | PaperSubscription => {
+  (state: AnyCheckoutState, currencyId?: Option<IsoCurrency>): SubscriptionProductFields => {
     const {
       billingPeriod,
       fulfilmentOption,
       productOption,
       orderIsAGift,
       product,
-      giftMessage,
-      giftDeliveryDate,
     } = state.page.checkout;
 
     const readerType = orderIsAGift ? Gift : Direct;
     if (product === DigitalPack) {
       return {
+        productType: 'DigitalPack',
         currency: currencyId || state.common.internationalisation.currencyId,
         billingPeriod,
         readerType,
-        giftMessage,
-        giftDeliveryDate,
       };
-    }
+    } else if (product === GuardianWeekly) {
+      return {
+        productType: 'GuardianWeekly',
```

Comment:
Add import and then - 
```suggestion
        productType: GuardianWeekly,
```